packages/create-mc-app/bin/create-mc-app.js

```diff
@@ -71,12 +71,18 @@ const packageManager = useYarn ? 'yarn' : 'npm';
 // TODO: we could check for min yarn/npm versions
 // See https://github.com/facebook/create-react-app/blob/0f4781e8507249ce29a9ac1409fece67c1a53c38/packages/create-react-app/createReactApp.js#L225-L254
 
-const templatePath = path.join(__dirname, '../templates', templateName);
+const templatePackageName = `@commercetools-frontend/application-template-${templateName}`;
+const modulePathForPackage = require.resolve
+  .paths(templatePackageName)
```

Comment:
The `require.resolve.paths` returns a list of location of `node_modules` that are checked by the node module resolution. For example:

<img width="692" alt="image" src="https://user-images.githubusercontent.com/1110551/50771324-9a002b80-128a-11e9-9dbe-20ec51fe397e.png">